backend/locust/locustfile.sample.py

```diff
@@ -0,0 +1,19 @@
+import time
+from locust import HttpUser, task, between
+
+class QuickstartUser(HttpUser):
```

Comment:
Do we still need this class? It looks like it's just a sample that was used to create the QuickstartSimpleReport file.

Reply:
This entire sample file is left for reference, to show how this can be expanded as we develop a definitive test suite. 😄 If we would prefer it to be purged, I can do so!